pages/common/gomi.md

```diff
@@ -0,0 +1,21 @@
+# gomi
+
+> Manage the trashcan.
+> See also: `trash`, `rm`.
+> More information: <https://github.com/babarot/gomi>.
+
+- Safely delete specific files or folders:
+
+`gomi {{path/to/file1 path/to/file2 path/to/folder1 path/to/folder2 ...}}`
+
+- Open an interactive menu to restore one or more files:
+
+`gomi {{[-b|--restore]}}`
+
+- Remove files older than a specific time ([d]ay, [w]eek, [m]onth, [y]ear):
```

Comment:
```suggestion
- Remove files from the trashcan older than a specific time ([d]ay, [w]eek, [m]onth, [y]ear):
```
This probably needs clarification. Otherwise someone might think it's talking about the current working directory.